VersionControlUI/src/DiffManager.cpp

```diff
@@ -102,9 +105,22 @@ DiffSetup DiffManager::initializeDiffPrerequisites(QString oldVersion, QString n
 	return diffSetup;
 }
 
+void DiffManager::clear()
+{
+	auto currentViewItem = Visualization::VisualizationManager::instance().
+			mainScene()->currentViewItem();
+
+	// remove overlays
+	currentViewItem->scene()->removeOverlayGroup(OVERVIEW_HIGHLIGHT_OVERLAY_NAME);
+	currentViewItem->scene()->removeOverlayGroup(OVERVIEW_ICON_OVERLAY_NAME);
+
+	Visualization::VisualizationManager::instance().mainScene()->removeOnZoomHandlers();
```

Comment:
Does this remove all of the handlers? What if someone else installed a handler for a different reason? Shouldn't there be a way to remove only the handlers that the diff installed? Before actually implementing this, let me know how much it would be.